src/data/stakeholdersData.json

```diff
@@ -5,18 +5,18 @@
o make their own resource more discoverable, increasing exposure and credit outside of their immediate community and ultimately promote adoption (learn how to add your resource to FAIRsharing, or claim it, at <a href='/new' class='underline-effect' target=blank>https://fairsharing.org/new</a>). A representative of a community standardisation initiative is best placed to describe the status of a standard(s) and to track its evolution. This can be done by creating an individual record (e.g., the DDI standard for social, behavioral, economic, and health data; <a href='https://doi.org/10.25504/FAIRsharing.1t5ws6' class='underline-effect' target=blank>https://doi.org/10.25504/FAIRsharing.1t5ws6</a>) or by grouping several records together in a collection (e.g., the <a href='/collection/HUPOPSI' class='underline-effect' target=blank>HUPO PSI standards</a> for proteomics and interactomics data). To achieve FAIR data, linked data models need to be provided that allow the publishing and connecting of structured data on the web. Similarly, representatives of a database or repository are uniquely placed to describe their resource, and to declare the standards you implement (e.g., the ICPSR archive of behavioural and social science research data that uses the DDI standard: <a href='https://doi.org/10.25504/FAIRsharing.y0df7m' class='underline-effect' target=blank>https://doi.org/10.25504/FAIRsharing.y0df7m</a>) or the Reactome knowledge base, ,<a href='https://doi.org/10.25504/FAIRsharing.tf6kj8' class='underline-effect' target=blank>https://doi.org/10.25504/FAIRsharing.tf6kj8</a>, which uses several standards in the <a href='/collection/ComputationalModelingCOMBINE' class='underline-effect' target=blank>COMBINE collection</a> for computational models in biology networks.\n\t The more adopted a resource is, the greater its visibility. For example, if your standard is implemented by a repository, these two records will be interlinked; thus if someone is interested in that repository they will see that your standard is used by that resource. If your resource is recommended in a data policy from a journal, funder or other organisation, it will also be searchable under a dedicated Recommendations (<a class='underline-effect' href='https://beta.fairsharing.org/search?isRecommended=true&page=1&searchAnd=false&fairsharingRegistry=database,standard'>https://beta.fairsharing.org/search?isRecommended=true&page=1&searchAnd=false&fairsharingRegistry=database,standard</a>) page.\n If you are the developer of a tool that requires access to the descriptions we provide on standards, databases, policies, as well as our collections, you can use the FAIRsharing API to power your system with our information. See our Licence (<a href='https://beta.fairsharing.org/licence' class='underline-effect'>https://beta.fairsharing.org/licence</a>) and API (<a href='https://beta.fairsharing.org/API_doc' class='underline-effect'>https://beta.fairsharing.org/API_doc</a>) pages for details.",
```

Comment:
```suggestion
        "content": "Standard developers/database curators can use FAIRsharing to explore what resources exist — and if they can be used or extended — in their area of interest, as well as to make their own resource more discoverable, increasing exposure and credit outside of their immediate community and ultimately promote adoption (learn how to add your resource to FAIRsharing, or claim it, at <a href='/new' class='underline-effect' target=blank>https://fairsharing.org/new</a>). A representative of a community standardisation initiative is best placed to describe the status of a standard(s) and to track its evolution. This can be done by creating an individual record (e.g., the DDI standard for social, behavioral, economic, and health data; <a href='https://doi.org/10.25504/FAIRsharing.1t5ws6' class='underline-effect' target=blank>https://doi.org/10.25504/FAIRsharing.1t5ws6</a>) or by grouping several records together in a collection (e.g., the <a href='/collection/HUPOPSI' class='underline-effect' target=blank>HUPO PSI standards</a> for proteomics and interactomics data). To achieve FAIR data, linked data models need to be provided that allow the publishing and connecting of structured data on the web. Similarly, representatives of a database or repository are uniquely placed to describe their resource, and to declare the standards you implement (e.g., the ICPSR archive of behavioural and social science research data that uses the DDI standard: <a href='https://doi.org/10.25504/FAIRsharing.y0df7m' class='underline-effect' target=blank>https://doi.org/10.25504/FAIRsharing.y0df7m</a>) or the Reactome knowledge base, <a href='https://doi.org/10.25504/FAIRsharing.tf6kj8' class='underline-effect' target=blank>https://doi.org/10.25504/FAIRsharing.tf6kj8</a>, which uses several standards in the <a href='/collection/ComputationalModelingCOMBINE' class='underline-effect' target=blank>COMBINE collection</a> for computational models in biology networks.\n\t The more adopted a resource is, the greater its visibility. For example, if your standard is implemented by a repository, these two records will be interlinked; thus if someone is interested in that repository they will see that your standard is used by that resource. If your resource is recommended in a data policy from a journal, funder or other organisation, it will also be searchable under a dedicated Recommendations (<a class='underline-effect' href='https://beta.fairsharing.org/search?isRecommended=true&page=1&searchAnd=false&fairsharingRegistry=database,standard'>https://beta.fairsharing.org/search?isRecommended=true&page=1&searchAnd=false&fairsharingRegistry=database,standard</a>) page.\n If you are the developer of a tool that requires access to the descriptions we provide on standards, databases, policies, as well as our collections, you can use the FAIRsharing API to power your system with our information. See our Licence (<a href='https://beta.fairsharing.org/licence' class='underline-effect'>https://beta.fairsharing.org/licence</a>) and API (<a href='https://beta.fairsharing.org/API_doc' class='underline-effect'>https://beta.fairsharing.org/API_doc</a>) pages for details.",
```